synapse/federation/transport/client.py

```diff
@@ -50,7 +50,7 @@ def get_room_state(self, destination, room_id, event_id):
         )
 
     @log_function
-    def get_event(self, destination, event_id):
+    def get_event(self, destination, event_id, timeout=None):
         """ Requests the pdu with give id and origin from the given server.
```

Comment:
Some doc string explaining how the timeout is used and what units it is in, and what happens if the timeout is reached might be nice.